README.md

```diff
@@ -14,15 +14,21 @@ Turn complex GraphQL queries into optimized database queries.
 pip install graphql-compiler
 ```
 
+## Quick Overview 
+
+The GraphQL compiler turns read-only queries written in GraphQL syntax to different 
```

Comment:
Nice summary! What do you think about flipping the order of these two sentences? That way, we start with the answer to "why should I care?" rather than "how does it work?", which might be a bit more user-friendly.